README.md

```diff
@@ -188,7 +182,7 @@ The following APIs are available for DoctorKafka:
     - List Cluster
     - Maintenance Mode
 
-Detailed description of APIs can be found [docs/APIs.md](docs/APIs.md)
```

Comment:
keep the `docs` prefix to make it clear to the reader that the .md file is under `docs` directory?